packages/aws-cdk-lib/region-info/build-tools/generate-static-data.ts

```diff
@@ -12,6 +12,7 @@ import {
   RULE_CLASSIC_PARTITION_BECOMES_OPT_IN,
 } from '../lib/aws-entities';
 import { Default } from '../lib/default';
+import { Runtime } from '../../aws-lambda';
```

Comment:
Here's a risk of the current repo layout: dependencies that shouldn't be.

`region-info` is a fundamental library and shouldn't depend on construct libraries. Can you hardcode the strings here?

And also: can you think of a mechanism to help us prevent these kinds of circular dependencies? I've been on a project where nothing was enforced and the dependencies were the most goddamn exquisite plate of spaghetti you've ever seen, and when we finally wanted to pick stuff apart it was extremely hard.

Reply:
since region-info is a duplicated package, maybe we can enforce it during build of the @aws-cdk/region-info version (which is copied from here). The linter will enforce that all modules that it imports are it its dependencies and we can add a lint rule for those duplicated packages that they can't take any internal dependencies. Meaning nothing within our repository?

Semi related as the new build mechanism for the @aws-cdk/region-info version https://github.com/aws/aws-cdk/pull/24929

Reply:
removed this dependency for now though.